src/lib/configs/project/fluence.ts

```diff
@@ -496,6 +501,39 @@ const configSchemaV3: JSONSchemaType<ConfigV3> = {
   },
 } as const;
 
+const configSchemaV3: JSONSchemaType<ConfigV3> = configSchemaV3Obj;
```

Comment:
There's 400 (!!!) lines between `configSchemaV0` and `configSchemaV3` declarations. That's wild!

Let's group consts at the top, then types, then functions.

Also I suggest to separate this file into several, it's way too big.

Reply:
I really would suggest configs refactoring to a separate PR